lib/osv/vulnerability.proto

```diff
@@ -55,13 +55,9 @@ message Package {
   // ecosystem (e.g. the npm package name). For C/C++ projects integrated in
   // OSS-Fuzz, this is the name used for the integration.
   string name = 1;
-  // Required. The ecosystem for this package. For vulnerabilities from OSS-Fuzz
-  // that do not have a package ecosystem (e.g. C/C++ packages), this is
-  // "OSS-Fuzz".
-  // Other valid values are:
-  //   - "Go" for Go modules.
-  //   - "PyPI" for Python packages.
-  //   - "npm" for NPM packages.
+  // Required. The ecosystem for this package. 
+  // For the complete list of valid ecosystem names, see
+  // <https://ossf.github.io/osv-schema/#affectedpackage-field>.
```

Comment:
Why the brackets?

Reply:
This is part of the Markdown spec for explicitly writing links: https://daringfireball.net/projects/markdown/syntax#autolink